charts/prl-ccd-definitions/Chart.yaml

```diff
@@ -4,7 +4,7 @@ name: prl-ccd-definitions
 version: 0.0.13
 dependencies:
   - name: nodejs
-    version: ~2.4.8
+    version: ~2.4.13
```

Comment:
can you please make it precisely 2.4.12?

Reply:
Just for a trail run of the prs i have upgraded after it is failed for 2.4.12. I will revert this change